p2p/simulations/adapters/exec.go

```diff
@@ -104,7 +104,7 @@ func (e *ExecAdapter) NewNode(config *NodeConfig) (Node, error) {
 	conf.Stack.P2P.NoDiscovery = true
 	conf.Stack.P2P.NAT = nil
 	conf.Stack.NoUSB = true
-	conf.Stack.Logger = log.New("node.id", config.ID.String())
+	//conf.Stack.Logger = log.New("node.id", config.ID.String())
```

Comment:
This has been removed on the latest `p2p-simulations-fixes` branch, rebasing `swarm-network-rewrite` on top of that should mean this change is not necessary.